### PR TITLE
Fixes the angle parsing issue #794 for majority of archives

### PR DESCRIPTION
**Pull request by AyushYadav**
@bsipocz  @keflavich Fix for #794 the angle parsing. Uses `astropy` for parsing rather than parsing it itself and uses wrappers in a better manner.

### CODE REVIEW

**Comment by bsipocz** [comment]
@AyushYadav - My thinking was in the opposite direction with #794. 
E.g ``commons.parse_radius`` is now a one liner (it was originally used to work around <0.3 astropy incompatibilies), and thus can be removed and ``coord.Angle()`` be used instead.

**Comment by AyushYadav** [comment]
@bsipocz My thinking was to make it all route through one wrapper of `utils.commons` so if at all some things changes from the Astropy end, this would only mean one line change in future. That was my idea behind using `commons.parse_radius()` for this one. Let me know if I should use `coord.Angle()` directly.

**Comment by bsipocz** [comment]
Astropy is unlikely making any backward incompatible changes to Angle in the near future, and as mentioned before it was there to support ancient astropy versions. So I would vote for getting rid of ``commons.parse_radius()`` altogether. Any veto to that @keflavich?

**Comment by AyushYadav** [comment]
@bsipocz Should I also remove the `commons.parse_radius()` fucntion altogether from the  [utils.commons](https://github.com/astropy/astroquery/blob/master/astroquery/utils/commons.py) section ? So as to get it remove it altogether.

**Comment by bsipocz** [comment]
Let's wait for @keflavich's opinion, but as I said my vote is on yes. 
Refactoring and then removing code though may seem weird at first, but is a very good thing.

**Comment by keflavich** [comment]
Hmm, I follow @AyushYadav's logic that it's nice to have one single place to resolve any future issues involving astropy angles or angle parsing in general.  But in this case it's such a simple function that... I agree with @bsipocz.  Let's get rid of that extra code.

File: astroquery/ned/core.py [hunk 1]
```diff
@@ -704,6 +704,23 @@ def _parse_result(self, response, verbose=False):
 Ned = NedClass()
 
 
+def _parse_radius(radius):
```
Comment (bsipocz) [inline astroquery/ned/core.py]: Any reason for adding this back?

Comment (AyushYadav) [inline astroquery/ned/core.py]: There are test units that are using this. I was hoping this had some functionalities for these two particular API's so I thought better keep them back. Working on it.

Comment (AyushYadav) [inline astroquery/ned/core.py]: @bsipocz I could only find two modules failing, `nvas` and `ned` , is there anything else that need to be edited before I submit another commit. I have locally fixed the issue for these two.

**Comment by AyushYadav** [comment]
@bsipocz @keflavich Build success. Any ideas about how to increase the coverage?

File: astroquery/ned/core.py [hunk 2]
```diff
@@ -718,7 +718,7 @@ def _parse_radius(radius):
         The value of the radius in arcminutes.
 
     """
-    return coord.Angle(radius).to('arcmin').value
+    return coord.Angle(radius).arcmin
```
Comment (bsipocz) [inline astroquery/ned/core.py]: @AyushYadav - Could you remove these functions (both from here, and from nvas)? Obviously then you can remove their tests from the test files, too so nothing will fail in there either.

Comment (AyushYadav) [inline astroquery/ned/core.py]: @bsipocz Done. Build in progress.

**Comment by AyushYadav** [comment]
@bsipocz The changes you asked are done. The wrappers around the ``_parse_radius()`` are removed and also their test units. The travis and appveyor build were successful. Can this be merged now?

**Comment by bsipocz** [comment]
This looks good, thanks @AyushYadav.

**Pull request merged**